test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_mnist.py

```diff
@@ -239,52 +249,51 @@ def test_smdataparallel_smmodelparallel_mnist(sagemaker_session, instance_type,
     image_cuda_version = get_cuda_version_from_tag(ecr_image)
     if Version(image_framework_version) < Version("2.3.1") or image_cuda_version != "cu110":
         pytest.skip("SMD Model and Data Parallelism are only supported on CUDA 11, and on TensorFlow 2.3.1 or higher")
-    smmodelparallel_path = os.path.join(RESOURCE_PATH, 'smmodelparallel')
+    smmodelparallel_path = os.path.join(RESOURCE_PATH, "smmodelparallel")
     test_script = "smdataparallel_smmodelparallel_mnist_script_mode.sh"
-    estimator = TensorFlow(entry_point=test_script,
-                           role='SageMakerRole',
-                           instance_count=1,
-                           instance_type=instance_type,
-                           source_dir=smmodelparallel_path,
-                           sagemaker_session=sagemaker_session,
-                           image_uri=ecr_image,
-                           framework_version=framework_version,
-                           py_version='py3')
+    estimator = TensorFlow(
+        entry_point=test_script,
+        role="SageMakerRole",
+        instance_count=1,
+        instance_type=instance_type,
+        source_dir=smmodelparallel_path,
+        sagemaker_session=sagemaker_session,
+        image_uri=ecr_image,
+        framework_version=framework_version,
+        py_version="py3",
+    )
 
     estimator = _disable_sm_profiler(sagemaker_session.boto_region_name, estimator)
 
     estimator.fit()
 
 
-def _assert_checkpoint_exists_v2(region, model_dir, checkpoint_number):
+def _assert_checkpoint_exists_v1(region, model_dir, checkpoint_number):
```

Comment:
the v1, no v thing is still confusing. What does new/old mean? what makes the distinction?